src/main/java/sc/fiji/io/Open_MRC_Leginon.java

```diff
@@ -229,6 +230,12 @@ public void run(final String arg) {
 		ImageStack stack = imp.getStack();
 		setStack(imp.getTitle(),stack);
 		setCalibration(imp.getCalibration());
+
+		for (int i=1; i<=stack.getSize(); ++i) {
```

Comment:
Please add an [NB comment](http://imagej.net/Coding_style#Javadoc_and_comments) here, to the effect of your commit message above. That way, anyone reading the source will understand exactly why we do this.